client/state/preferences/schema.js

```diff
@@ -13,20 +13,6 @@ export const remoteValuesSchema = {
 		mediaModalGalleryInstructionsDismissed: {
 			type: 'boolean',
 		},
-		'google-my-business-dismissible-nudge': {
-			type: 'object',
-			'^[1-9]+$': {
-				type: 'array',
-				items: {
-					type: 'object',
-					properties: {
-						dismissedAt: { type: 'number', minimum: 0 },
-						type: { type: 'string', enum: [ 'dismiss' ] },
-					},
-					required: [ 'dismissedAt', 'type' ],
-				},
-			},
-		},
```

Comment:
This schema file also defines the `upwork-dismissible-banner` pref, which should be also removed in this PR.

Reply:
I thought the same and was going to remove it when working on this PR, but then I noticed that it's still used by the `isUpworkBannerDismissed` selector and the `dismissBanner` action creator, and they're still used by the `blocks/upwork-banner` component.